src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -59,11 +81,15 @@ object Task {
   /**
     * Contains information about the status of a launched task including timestamps for important
     * state transitions.
+    *
+    * @param stagedAt Despite its name, stagedAt is set on task creation and before the TASK_STAGED notification from
```

Comment:
If using a fresh approach, I would call it `launchedAt`.

If we rename it now, it might be confusing because in the API it is called `stagedAt`. What do you think?